cross/tvheadend/Makefile

```diff
@@ -11,7 +11,12 @@ PKG_DIR = $(PKG_NAME)-$(PKG_GIT_HASH)
 DEPENDS  = cross/openssl cross/gnutls
 DEPENDS += cross/uriparser cross/pcre2 cross/libiconv cross/fontconfig
 DEPENDS += cross/libhdhomerun cross/libdvbcsa cross/dvb-apps cross/pngquant
+DEPENDS += cross/libwebsockets
```

Comment:
@th0ma7 I have moved the `cross/libwebsockets` dependency from curl to tvh, assuming tvh uses libwebsockets directly.

`cross/libwebsockets` does not add any feature to curl. Does tvh need webservice support in curl/libcurl ?
If so, cross/libwebsockets can be removed at all, and tvh can build curl with `--enable-websockets` by adding `websockets` to `BUILD_CURL_WITH` variable.
But this is an experimental feature of curl.

Reply:
I took the time to dig a little on this and I can't find anything in the code pertaining to libwebsockets library usage.  TVH does however create websocket but seems to be doing so on its own.  As such I don't believe libwesockets is relevant anymore.